@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -204,7 +204,7 @@
                     </td>
                     <td>
                       <UiInput
-                        v-model="networkInterface.macAddress"
+                        v-model="networkInterface.macAddress!"
```

Comment:
Why do you use `!` here?
If `macAddress` cannot be undefined, so maybe the type must be fixed